torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16378,6 +16378,11 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
            supports_forward_ad=True,
            supports_fwgrad_bwgrad=True,
            sample_inputs_func=sample_inputs_take,
+           # eager gather() does not support negative indices but compiled
+           # gather does.
+           skips=(
+               DecorateInfo(unittest.expectedFailure),
```

Comment:
This is skipping all tests. You should just skip the failing tests. See how other OpInfos do it